docs/installation/changelog.md

```diff
@@ -25,20 +28,29 @@
 
 - Optimize logging developer messages; [#3057](https://github.com/ExpressionEngine/ExpressionEngine/issues/3057)
 - Added keyboard shortcut for saving Pro Variables
+- Added icon for video file
+- Added icon for doc file type
+- Installer / Updated now respects default theme colors
+- Remove "blinks" into the drak on script loads
```

Comment:
Can we reword this one? I have no idea what is means, and I think others reading the changelog will also have no idea what is meant by it